src/Components/Calculator.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable  react/prefer-stateless-function */
```

Comment:
- [ ] Please remove all the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :smile: 

- [ ] I would like to recommend you to setup correctly your linters, especially ESLint please follow this [link](https://github.com/microverseinc/linters-config/tree/master/react-redux)